__tests__/seed.test.js

```diff
@@ -52,3 +52,46 @@ describe('GET /api', () => {
     })
 });
 
+describe('/api/articles/:article_id/comments', () => {
+    test("200: returns an array of comments for the given article_id", () => {
+        return request(app).get('/api/articles/3/comments')
+            .expect(200)
+            .then(({ body }) => {
+                expect(body.rows.length).toBe(2);
+                expect(body.rows[0]).toMatchObject(
+                    {
+                        comment_id: 11,
+                        body: 'Ambidextrous marsupial',
+                        article_id: 3,
+                        author: 'icellusedkars',
+                        votes: 0,
+                        created_at: "2020-09-19T23:10:00.000Z"
+                    },
+                )
+                body.rows.forEach((comment) => {
+                    expect(comment).toMatchObject({
+                        comment_id: expect.any(Number),
+                        body: expect.any(String),
+                        article_id: expect.any(Number),
+                        author: expect.any(String),
+                        votes: expect.any(Number),
+                        created_at: expect.any(String)
+                    });
+                })
+            })
+    });
+    test("404: responds with an error message if the article_id does not exist", () => {
+        return request(app)
+            .get('/api/articles/26/comments')
+            .expect(404)
+            .then(({ body }) => {
+                expect(body.msg).toBe("not found")
+            })
+    });
+    test('400: responds with an error message if passed an invalid article_id', () => {
+        return request(app).get('/api/articles/banana/comments')
+            .expect(400).then(({ body }) => {
+                expect(body.msg).toBe('bad request');
```

Comment:
Looking good, what happens if an article has no comments? What status code should that be?